index.html

```diff
@@ -304,6 +305,291 @@ <h2>Contact me</h2>
+                    <p>Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. Lorem Ipsum has been 
+                        the industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of type 
+                        and scrambled it to make a type specimen book.</p>
+                    <p>Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. Lorem Ipsum has been 
+                        the industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of 
+                        type and scrambled it 1960s with the releaLorem 
+                        Ipsum is simply dummy text of the printing and 
+                        ypesetting industry. Lorem Ipsum has been the 
+                        industry's standard dummy text ever since the 
+                        1500s, when an unknown printer took a galley of 
+                        type and scrambled it 1960s with the releorem 
+                        Ipsum is simply dummy text of the printing and 
+                        typesetting industry. Lorem Ipsum han printer 
+                        took a galley of type and scrambled it 1960s with 
+                        the releawn printer took a galley of type and 
+                        scrambled it 1960s with the releaLorem Ipsum is 
+                        simply dummy text of the printing and typesetting 
+                        industry. Lorem Ipsum has been the industry's 
+                        standard dummy text ever since the 1500s, when 
+                        an unknown printer took a galley of type and 
+                        scrambled it 1960s with the relea</p>
+                </div>
+                <div class="modal-dir">
+                <div class="modal-tag">
+                    <ul>
+                        <li> <div class="tag">html</div></li>
+                        <li> <div class="tag1">css</div></li>
+                        <li> <div class="tag2">javascript</div></li>
+                    </ul>
+                    <ul>
+                        <li><div class="tag">github</div></li>
+                        <li><div class="tag1">Ruby</div></li>
+                        <li><div class="tag2">Bootstrap</div></li>
+                    </ul>
+                </div>
+                <img class="line" src="asset/Vector4.svg" alt="line">
+                <div class="modal-btn">
+                    <button type="button" class="modal-project">See Live
+                        <img src="asset/Icon.svg" alt="logo">
+                    </button>
+                    <button type="button" class="modal-project">See Source
+                        <img src="asset/Vector.svg" alt="git_logo">
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] For this project you are expected to create these four project objects automatically out of an array of sample projects to follow the DRY principle. Kindly create them via JavaScript and out of an array of objects
![javaScript requirements](https://user-images.githubusercontent.com/69858269/169398715-1eb13c6b-1915-46e1-bb57-3c6715b2cb19.PNG)

Reply:
rectified